pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -135,6 +137,38 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage, error) {
 		DeleteStrategy:          bindingRESTStrategies,
 		EnableGarbageCollection: true,
 
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "Instance", Type: "string"},
+				{Name: "Secret", Type: "string"},
```

Comment:
This should be `SecretName`

Reply:
OK, I've changed this to `Secret-Name` so it gets printed as `SECRET-NAME`, but I think a simple `SECRET` is perfectly fine. This is just the column header.